CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 
 ### New Features
 
+* You can now easily create a new config file using `swiftgen config init`. This will create an example and commented config file and open it to let you edit it to your needs.  
```

Comment:
I'd add a small comment that it doesn't take the user's project into account in any way right now (but might at some later point).

Reply:
done.